src/NzbDrone.Core/ImportLists/Simkl/SimklAPI.cs

```diff
@@ -66,4 +73,10 @@ public class SimklTvSyncActivityResource
     {
         public DateTime All { get; set; }
     }
+
+    public class AnimeType
```

Comment:
I wonder is this is could be conflicting with the `AnimeType` on `SimklSeriesResource`, could try renaming one of them. This should also be converted to an enum.

Reply:
Okay, figured it out, the wrong JSON deserialization is being used in `SimklParser`,

```
var jsonResponse = STJson.Deserialize<SimklResponse>(_importResponse.Content);
```

should be 

```
var jsonResponse = Json.Deserialize<SimklResponse>(_importResponse.Content);
```

Reply:
This did the trick and AnimeType works just fine now. Thanks for looking into it for us!